src/poetry/packages/locker.py

```diff
@@ -412,10 +412,7 @@ def _dump_package(self, package: Package) -> dict[str, Any]:
             for name, deps in sorted(package.extras.items()):
                 # TODO: This should use dep.to_pep_508() once this is fixed
                 # https://github.com/python-poetry/poetry-core/pull/102
```

Comment:
Is this comment still relevant?

Reply:
I will remove it. `to_pep_508()` appends markers that are probably not required in that place. At least not something like `extra == "foo"`